### ISSUE TITLE
"Button" error on Web

### ISSUE THREAD

**Issue by MeromTh** [tamagui/tamagui#2893]
### Current Behavior

```
React does not recognize the `textAlign` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `textalign` instead.
```
cause by https://github.com/tamagui/tamagui/commit/6a1e1b11493594717822e9ab09bb42ff8026b2a6

### Tamagui Version

```markdown
1.110.1
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
- Clone this https://github.com/MeromTh/tamagui-bug
- run with `npm run web`
- error from index on the first visit
```

**Comment by natew** [comment]
I have a fix for this, will push it today.

**Comment by natew** [comment]
Let me know if 1.110.2 fixes it via https://github.com/tamagui/tamagui/commit/5b699ec7ab08d0e9b47082e3f0555c0e4cc24837

**Comment by michaelzadra1** [comment]
I just upgraded to 1.110.2, and am still seeing the warning in my browser console: 

`Warning: React does not recognize the `textAlign` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `textalign` instead. If you accidentally passed it from a parent component, remove it from the DOM element.`

**Comment by nakshatrasinghh** [comment]
I see this issue as well now! Any quick fix for this?

**Comment by nakshatrasinghh** [comment]
> Let me know if 1.110.2 fixes it via [5b699ec](https://github.com/tamagui/tamagui/commit/5b699ec7ab08d0e9b47082e3f0555c0e4cc24837)

Latest version has the same issue as well!

**Comment by simonszalai** [comment]
+1

**Comment by natew** [comment]
Fix going out now in 1.110.3
